src/main/java/duke/Task.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    private String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public boolean getDoneStatus() {
```

Comment:
Shouldn't boolean methods sound like booleans? Any reason why you didn't do something like `isDone` or `hasCompleted`?